pkgs/applications/editors/glow/default.nix

```
@@ -3,19 +3,28 @@
 , fetchFromGitHub
 , installShellFiles
 , stdenv
+, fetchpatch
 }:
+
 buildGoModule rec {
   pname = "glow";
-  version = "1.5.0";
+  version = "1.5.1";
 
   src = fetchFromGitHub {
     owner = "charmbracelet";
     repo = "glow";
     rev = "v${version}";
-    sha256 = "sha256-CI0S9XJtJQClpQvI6iSb5rcHafEUwr2V6+Fq560lRfM=";
+    hash = "sha256-12UziCf3BO1z+W02slNCCvXhIkvZuVgXk++BdHG3gDI=";
   };
 
-  vendorHash = "sha256-2QrHBbhJ04r/vPK2m8J2KZSFrREDCc18tlKd7evghBc=";
+  vendorHash = "sha256-xxFC87t12bZKea9Snscul+xx8IGFAcoIr9Z8wxHL7nM=";
+
+  # Remove whenever https://github.com/charmbracelet/glow/pull/496 is merged and a release with it is available
```

```suggestion
  # Remove whenever a release with it is available
```
The commits I suggested have already been merged into the master.

---

fixed in https://github.com/NixOS/nixpkgs/pull/257931/commits/4f1206ff05bd5601e6f05439cbc216915e0e52f9